### ISSUE TITLE
[BUG] does anyone know why SimplePie might parse this RSS feed as empty?

### ISSUE THREAD

**Issue by AlexHarrowell** [simplepie/simplepie#626]
DESCRIPTION: WordPress's fetch_feed() function, which is a wrapper around SimplePie, consistently parses the following non-empty RSS feed as empty:

https://feeds.pinboard.in/rss/u:yorksranter/

This began with a WP update early last year

#### Steps to Reproduce

Try to load an RSS feed for display on the sidebar of my blog. Debugging of the WP install showed that nothing is being returned from SimplePie to fetch_feed, although requesting the feed from a browser with the SimplePie user agent set returns the feed.

#### Expected Behavior

The RSS feed would be retrieved, parsed, and entries displayed on the relevant column of my website

#### Actual Behavior

Nothing.

#### Possible Solutions

Basically no idea. I suspect Pinboard of behaving strangely with non-browser or non-app clients but I have yet to prove it.

#### Environment

Ubuntu Linux 16.04.3 LTS, PHP 7.0.33, Apache 2.4.18, WordPress 5.2.3

**Comment by skyzyx** [comment]
What happens when you enable error reporting? `error_reporting(-1);`

**Comment by rongybika** [comment]
Hi,

I have some similar issue using localhost. Some pages works some not (so I'm not geting always error). But I noticed this cURL error message:

public 'error' => string 'cURL error 60: SSL certificate problem: unable to get local issuer certificate'

Same for http or https feed links.

**Comment by rongybika** [comment]
I managed to resolve my issue modifying curl.cainfo in PHP.ini to be like this:
curl.cainfo="c:\xampp\php\extras\ssl\curl-ca-bundle.crt"

**Comment by AlexHarrowell** [comment]
I needed to install php-xml and chase up some dependencies, then wait for a cache timer to expire. Closing the ticket.
